xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -54,8 +54,7 @@
 
 // amcodec include
 extern "C" {
-#include <codec.h>
-#include <amports/amstream.h>
+#include <amcodec/codec.h>
```

Comment:
Is this now the final preferred place? If so we should finally nail this and change FindAMLcmake script so it searches codec.h at this place.

Reply:
Actually FindAML.cmake looks for amcodec/codec.h: https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindAML.cmake#L16-L25

Reply:
👍 ok, great, seems that it was already changed in the past.